src/main/java/in/twizmwaz/cardinal/module/modules/bossBar/FakeWither.java

```diff
@@ -40,10 +41,6 @@ public float getMaxHealth() {
         return maxHealth;
     }
 
-    public void setMaxHealth(float max) {
-        maxHealth = max;
-    }
-
```

Comment:
@Alfalik What was the point of moving this method? Its purely an aesthetic decision at this point and shouldn't effect your PR in any way. Same for the movement of the variables above, which were structured that way for a reason. Public variables were on top and the private ones were on the bottom, there is no point in changing this.